lib/administrate/view_generator.rb

```diff
@@ -21,7 +21,8 @@ def copy_resource_template(template_name)
     end
 
     def resource_path
-      args.first.try(:underscore).try(:pluralize) || "application"
+      return "application" if args.none? || args.first == "application"
+      args.first.try(:underscore).try(:pluralize)
```

Comment:
[Our style guide](https://github.com/thoughtbot/guides/tree/master/style/ruby) says to avoid conditional modifiers (lines that end with conditionals). Can we re-phrase this as:

```ruby
if args.none? || args.first == "application"
  "application"
else
  args.first.underscore.pluralize
end
```

Also, I think we can ditch the `try()` methods, because we now know that args.first is defined at this point.